### PR TITLE
confirm before deleting config with modal

### LINKED ISSUE #387: Prompt for confirmation before deleting configs

### PR DESCRIPTION
**Pull request by abemassry**
closes #387 
image

### CODE REVIEW

**Comment by eatonphil** [comment]
Looks like we've got a modal subfolder now. You're moving this modal out of it?

File: src/linodes/linode/settings/components/ConfigPanel.js [hunk 1]
```diff
@@ -38,7 +40,19 @@ function configContent(linode, configs, dispatch) {
...
+                  dispatch(showModal('Confirm deletion',
+                    <ConfirmModal
+                      buttonText="Delete config"
+                      onOk={async () => {
+                        await dispatch(linodes.configs.delete(linode.id, config.id));
+                        dispatch(hideModal());
+                      }}
+                      onCancel={() => dispatch(hideModal())}
+                    >
+                      Are you sure you want to delete the config?
```
Comment (eatonphil) [inline src/linodes/linode/settings/components/ConfigPanel.js]: s/the/this/

**Comment by eatonphil** [comment]
What's the deal with this ConfirmModal vs the existing ConfirmModal?

**Comment by abemassry** [comment]
There was some merge conflict resolution related thing.

**Comment by abemassry** [comment]
@eatonphil asked for a filename change

**Pull request merged**